frontend/src/components/partners/partnersProgresBar.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { FormattedMessage } from 'react-intl';
+
+import messages from './messages';
+import { ProgressBar } from '../progressBar';
+
+export default function PartnersProgresBar({
+  totalData,
+  percentValidated,
+  label,
+  value,
+  small = true,
+}: Object) {
+
+  return (
+    <>
+    
```

Comment:
unnecessary space

Reply:
@royallsilwallz code styling improvements like this one will be on our side.